### PR TITLE
Deploy 26-06-2023

### PR DESCRIPTION
**Pull request by attiks**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -8,7 +8,7 @@ root = true
 [*]
 end_of_line = LF
 indent_style = space
-indent_size = 2
+indent_size = 4
```
Comment (cafuego) [inline .editorconfig:11]: WHAT?!!

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -189,7 +210,7 @@ jobs:
 
       - name: Slack Success Notification
         id: slack_success
-        if: success()
+        if: ${{ !env.ACT && success() }}
         uses: slackapi/slack-github-action@v1.23.0
         with:
           channel-id: '${{ secrets.SLACK_CHANNEL }}'
```
Comment (cafuego) [inline .github/workflows/run-tests.yml:216]:
```suggestion
          channel-id: '${{ vars.SLACK_CHANNEL }}'
```

File: .github/workflows/run-tests.yml [hunk 3]
```diff
@@ -216,7 +237,7 @@ jobs:
 
       - name: Slack Failure Notification
         id: slack_failure
-        if: failure()
+        if: ${{ !env.ACT && failure() }}
         uses: slackapi/slack-github-action@v1.23.0
         with:
           channel-id: '${{ secrets.SLACK_CHANNEL }}'
```
Comment (cafuego) [inline .github/workflows/run-tests.yml:243]:
```suggestion
          channel-id: '${{ vars.SLACK_CHANNEL }}'
```

**Review by cafuego — APPROVED** [review]
Grudgingly approved. A tab stop of 4 is terrible, end of the world stuff.

File: .editorconfig (hunk 1 above)
Comment (attiks) [reply .editorconfig:11]: I know ;-)

**Pull request merged**